src/components/FindABook/FindABook.jsx

```diff
@@ -0,0 +1,17 @@
+import '../BookCard/BookCard.css';
+import BookCard from '/src/components/BookCard/BookCard.jsx';
+import booksData from '/src/data/booksData.js';
+
+
+export default function FindABook({
```

Comment:
We already have a component named FindABook in the pages folder. Why do we need another one? Which one should be removed? Alternatively, if they are separate components, let's name them differently so we know what they are doing.

Reply:
this was from the beginning for testing and forgot to delete it